requirements.txt

```diff
@@ -1,7 +1,7 @@
-git+https://github.com/fastforwardlabs/cmlbootstrap#egg=cmlbootstrap
 pandas
 joblib
 scikit-learn 
 xgboost 
 numpy
 flask
+seaborn
```

Comment:
This is 100% my bad but could you pin dependencies here? 

(I think I  accidentally removed them when I upgraded this amp awhile ago)

pandas==1.3.5
joblib==1.1.0
scikit-learn==1.0.2 
xgboost==1.6.1 
numpy==1.19.4
Flask==2.2.1
seaborn==0.11.2

Reply:
Do you want me to pin these exact ones or get a fresh latest set that works?

Reply:
Hehe, these are the latest versions, committed the pins.